### ISSUE TITLE
CallKit informing multiple payloads bug

### ISSUE THREAD

**Issue by Romulomga** [twilio/voice-quickstart-ios#306]
### Description

Trying a voice call from phone A to phone B, phone B receives two voip pushes.

With iOS 13 restriction, I need to inform CallKit by doing twice:

`callKitProvider.reportNewIncomingCall(with: uuid, update: callUpdate)`

After reporting a new incoming call twice, CallKit starts but when I answer, the call terminates with call canceling.

### Steps to Reproduce

1. Call phone B using phone A
2. Report to CallKit the voip pushes received in Phone B
3. Try to answer
4. Twilio delegate return cancel call

#### Expected Behavior

Answer voice call in Phone B

#### Actual Behavior

Twilio delegate return call canceled

#### Reproduces How Often

Every time

#### Voice iOS SDK

[e.g. 5.1.1  via CocoaPods]

#### Xcode

[e.g. 11.3.1]

#### iOS Version

[e.g. 13.3.1] both phones

#### iOS Device

[e.g. iPhone SE]
[e.g. iPhone X]

**Comment by bchen-twilio** [comment]
Hi @Romulomga 

Thanks for reporting the issue. Can you provide us a Twilio Call SID which you experienced the issue?

-bobie

**Comment by Romulomga** [comment]
Call sid: `CA10708f11f8492bf0c0c7712ac7301dfa`

**Comment by bchen-twilio** [comment]
Hi @Romulomga 

Thanks for providing the Call SID.
I've checked the records and did see that two notifications were sent to the device. After some cross reference I found that two notification bindings were created for this identity with this device, but one with capitalized characters while the other one with lower-cased characters. This is the reason Twilio treated the two as different bindings.

This is definitely something either the SDK or the Twilio Notify service can fix in the future, but in the meantime to fix this, I'd recommend that you unregister the identity using both the upper-cased and the lower-cased device token (the one ends with `7C908842`), and register again. This should resolve the multiple notifications issue at the moment.

Hope this helps.
Sorry for the inconvenience.

-bobie
